h/static/scripts/vendor/annotator.js

```diff
@@ -622,7 +622,7 @@
       }
       nr = {};
       if (r.startOffset > 0) {
-        if (r.start.nodeValue.length > r.startOffset) {
+        if (!r.start.nextSibling || r.start.nodeValue.length > r.startOffset) {
```

Comment:
That is right, the selector does end up at the empty last node. However, this is ok because our anchoring process seems to be filtering out empty nodes as well. It will not wrap the empty node with spans. So it's really a non-issue right now